### DISCUSSION TITLE
Obs re-stream help

### DISCUSSION THREAD

**Discussion by chacinpradaj** [bluenviron/mediamtx#4064] [Questions]
### Question

I tried to set it up with an incoming Obs rtmp stream 
 Obs connect using rtmp://192.168.1.12  streamkey:obs and it successfully connects 
It is not hosted on the same computer as Obs

here is my config, I merge different solutions according to discussion and the readme.

paths:
  mypath:
    source: rtmp://127.0.0.1:5000/obs
    runOnReadyRestart: true
    runOnReady: sh -c "ffmpeg -i rtmp://127.0.0.1:5000/obs -c copy -f flv rtmp://a.rtmp.youtube.com/live2/youtube-key \
                      ffmpeg -i rtmp://127.0.0.1:5000/obs -c copy -f flv rtmp://ber.contribute.live-video.net/app/live_1219968844_key\
                      ffmpeg -i rtmp://127.0.0.1:5000/obs -c copy -f flv rtmps://live-api-s.facebook.com:443/rtmp/FACEBOOK_STREAM_KEY & wait"
                      
any help is welcome, and if I missed anything on the readme let me know.          
Thank you.

**Comment by aler9** [comment]
The `runOnReady` command is wrong, you are reading twice from the same source, the first time with `source: rtmp://127.0.0.1:5000/obs` and the second with `runOnReady: sh -c "ffmpeg -i rtmp://127.0.0.1:5000/obs`. This doesn't make sense.

Instructions on how to send streams from OBS to MediaMTX are in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#obs-studio

Instructions on how to send streams from MediaMTX to external servers (like YouTube, Facebook, etc) are in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#forward-streams-to-other-servers
